stacks/index.js

```diff
@@ -21,7 +22,7 @@ export default function (app) {
   app.setDefaultFunctionProps({
     runtime: 'nodejs16.x',
     environment: {
-      NODE_OPTIONS: "--enable-source-maps",
+      NODE_OPTIONS: "--enable-source-maps --experimental-fetch",
```

Comment:
https://github.com/web3-storage/fr32-sha2-256-trunc254-padded-binary-tree-multihash relies on `globalThis.fetch` and we will need to rely on this until [we can upgrade SST here to bump node version](https://github.com/web3-storage/w3infra/pull/152)

Reply:
ACK, lets get that upgrade done soon!